PxCs/Data/Vob/PxVobNpcData.cs

```diff
@@ -1,10 +1,81 @@
 ﻿using System;
+using System.Numerics;
 
 namespace PxCs.Data.Vob
 {
     [Serializable]
     public class PxVobNpcData : PxVobData
     {
+        public class PxVobNpcTalentData
```

Comment:
Why are these inner classes?
Is it for readability? (As we normally don't use inner classes)

Reply:
These inner classes are used only inside this class, and yeah, for readability